gapis/api/vulkan/state_rebuilder.go

```diff
@@ -897,6 +897,16 @@ func (sb *stateBuilder) createDevice(d DeviceObjectʳ) {
 			),
 		).Ptr())
 	}
+	if !d.PhysicalDeviceProvokingVertexFeaturesEXT().IsNil() {
+		pNext = NewVoidᵖ(sb.MustAllocReadData(
+			NewVkPhysicalDeviceProvokingVertexFeaturesEXT(
+				VkStructureType_VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_PROVOKING_VERTEX_FEATURES_EXT,
+				pNext,
+				d.PhysicalDeviceProvokingVertexFeaturesEXT().ProvokingVertexLast(),
+				d.PhysicalDeviceProvokingVertexFeaturesEXT().TransformFeedbackPreservesProvokingVertex(),
+			),
+		).Ptr())
+	}
```

Comment:
I think you're missing a hunk in the state rebuilder to attach the `VkPipelineRasterizationProvokingVertexStateCreateInfoEXT` to pipelines that are rebuilt from state?

Reply:
That is one of things that I am concerned and will research today. 

When I do a MEC, I can see that pipeline has the extension property. But state reconstruction does not explicitly add it. I will look for how does that happen.